scripts/poll_spi_result

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Haven't tested this but shouldn't we translate this script to groovy? AFAIK the init scripts copied to /usr/share/jenkins/ref/init.groovy.d/ must be written in groovy to be executed, see [1]. Let me know if it works for you with bash, if not I can try to make the translation.

Thanks!

[1] https://wiki.jenkins-ci.org/pages/viewpage.action?pageId=70877249